### ISSUE TITLE
Renaming saves not working

### ISSUE THREAD

**Issue by jwidess** [paul-merkamp/LCBetterSaves#15]
When clicking the **rename** button on a save, nothing happens and the following **error** is thrown in the console. 

That being said I do have 42 different mods installed which **is probably** causing this issue as when I was testing this mod originally with fewer mods, and v45, it did work properly. So I'm not sure if its my mods or the v47 update.
My current Thunderstore profile code: **018ceb12-691b-b790-8601-8d9d625fa375**

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
RenameFileButton_BetterSaves.RenameFile () (at <cd58899513e64a26a5c92f9619b85964>:IL_0020)
UnityEngine.Events.InvokableCall.Invoke () (at <e27997765c1848b09d8073e5d642717a>:IL_0010)
UnityEngine.Events.UnityEvent.Invoke () (at <e27997765c1848b09d8073e5d642717a>:IL_0022)
UnityEngine.UI.Button.Press () (at <17de86d66286445fa95af59fa8fa644d>:IL_001C)
UnityEngine.UI.Button.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <17de86d66286445fa95af59fa8fa644d>:IL_0009)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <17de86d66286445fa95af59fa8fa644d>:IL_0007)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <17de86d66286445fa95af59fa8fa644d>:IL_0067)
UnityEngine.EventSystems.EventSystem:Update()
``` 

**To clarify I'm clicking the following button,**
![image](https://github.com/paul-merkamp/LCBetterSaves/assets/24278368/c6cf10e4-09ca-4577-9938-b7f2fb12e955)

**Comment by paul-merkamp** [comment]
This appears to be a v47 issue, working on a resolution now

**Comment by paul-merkamp** [comment]
Fixed with 1.7.0, devs renamed the panel containing the text field so my mod couldn't find it anymore
